cpmpy/expressions/globalfunctions.py

```diff
@@ -256,6 +256,8 @@ def __init__(self, arr, idx):
             raise TypeError("index cannot be a boolean expression: {}".format(idx))
         if is_any_list(idx):
             raise TypeError("For using multiple dimensions in the Element constraint, use comma-separated indices")
+        if is_int(idx): # prevents OR-Tools error
```

Comment:
If it is OR-Tools specific, it should be fixed in the OR-Tools interface before posting the constraint. Now the user will see another expression that he created when printing it.

I.e., print(`Element(arr, 42) <= 1337`) will print `arr[IV0] <= 1337`

Reply:
Right, definitely the better approach (I was unable to come up with).